addon/templates/components/polaris-drop-zone.hbs

```diff
@@ -1,57 +1,58 @@
-{{! template-lint-disable no-invalid-interactive }}
-
-{{#with (if label
-  (component "polaris-labelled"
-    id=id
-    label=label
-    labelHidden=labelHidden
-    action=labelAction
-  )
-  (component "wrapper-element"
-    tagName=""
-  )
-) as |wrapper|}}
-  {{#component wrapper}}
+{{#let (if @label
+  (component "polaris-labelled")
+  (component "wrapper-element" tagName="")
+) as |Wrapper|}}
+  <Wrapper
+    @id={{this.id}}
```

Comment:
Should this be `@id` instead of `this.id` since `id` is a public [attribute](https://github.com/smile-io/ember-polaris/blob/43ea1c80acb2379b0cd29a5f7d2aa11ca3a1220a/addon/components/polaris-drop-zone.js#L37)?

Reply:
yes indeed...good catch
fixed